fastlane_bot/config/constants.py

```diff
@@ -34,3 +34,16 @@
 SECTA_V3_NAME = "secta_v3"
 METAVAULT_V3_NAME = "metavault_v3"
 ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
+
+BLOCK_CHUNK_SIZE_MAP = {
+    "ethereum": 0,
+    "polygon": 0,
+    "polygon_zkevm": 0,
+    "arbitrum_one": 0,
+    "optimism": 0,
+    "coinbase_base": 0,
+    "fantom": 5000,
+    "mantle": 0,
+    "linea": 0,
+    "sei": 0,
```

Comment:
Depending on the RPC URL planned for use, this limit may need to change:
- For `https://evm-rpc.arctic-1.seinetwork.io/`, use 0
- For `https://rpc.mantle.xyz`, use 10000

Reply:
This will require more granular configuration. Something like having a separate config parameters for each RPC, which we should strive for anyway. Let's keep at 0 for now, as that will work for both RPCs. But let's keep in mind that we might need to change it when the mainnet is launched.